src/app/components/TransactionsTable/TransactionModal.tsx

```diff
@@ -172,13 +178,13 @@ export default function TransactionModal({
 }
 
 const Dt = ({ children }: { children: React.ReactNode }) => (
-  <dt className="basis-28 text-gray-400 dark:text-neutral-500 text-right">
+  <dt className="w-28 text-gray-400 dark:text-neutral-500 text-right">
     {children}
   </dt>
 );
 
 const Dd = ({ children }: { children: React.ReactNode }) => (
-  <dd className="flex-1 text-gray-800 dark:text-neutral-200 break-all whitespace-pre-wrap">
+  <dd className="text-gray-800 dark:text-neutral-200 break-words whitespace-pre-wrap overflow-hidden">
```

Comment:
break-words + overflow-hidden breaks long text "aaaaaaaaa...aaaaa" inside of words, but doesn't just break words in-between.